### ISSUE TITLE
diag_table

### ISSUE THREAD

**Issue by wenyuz** [NOAA-GFDL/AM4#49]
I was running the 2021.03 version but the run returned with a fatal error as it cannot save some variables in the diag table.

For example, there are "frac_soil" and "area_soil" in diag table, 
"land",    "frac_soil",  "frac_soil",  "land_static", "all", .false., "none", 2

but there are no such variables in the code 
( that is, grep -r "frac_soil" . returns nothing)

when I remove these terms in diag table, it returned with another error
FATAL from PE   548: diag_manager_mod::init_field_cell_measure: AREA field not in diag_table for field soil/soil_liq

I wonder if the diag table is outdated.

Thanks,

**Comment by thomas-robinson** [comment]
Where did you find the diag_table? Is it the diag table in the run directory of this repo?

**Comment by wenyuz** [comment]
Yes, it is from the run directory. It has "frac_soil" which is not found in src.

**Comment by wenyuz** [comment]
Hi Thomas,

I wonder if there is any update about the issue. Thanks!

Wenyu

**Comment by thomas-robinson** [comment]
I just ran the model using the input.nml and diag_table in the run directory of this repository https://github.com/NOAA-GFDL/AM4/tree/main/run , and it worked.  If you are using the current code base and the original AM4_run directory, you need to replace the input.nml and diag_table with the files in https://github.com/NOAA-GFDL/AM4/tree/main/run .
